package.json

```diff
@@ -9,42 +9,36 @@
     "angular": "^1.5.9",
     "angular-loader": "^1.5.9",
     "angular-route": "^1.5.9",
-    "html5-boilerplate": "0.0.1",
-    "shelljs": "^0.7.5"
+    "html5-boilerplate": "0.0.1"
   },
   "devDependencies": {
     "angular-mocks": "^1.5.9",
     "bower": "^1.7.7",
+    "cpx": "^1.5.0",
```

Comment:
I still consider this a non-dev dependency. We rely on it to copy the files to `app/lib/` even in "production". Alternatively, we could add `app/lib/` to the repository (but I am not a big fan of that :smiley:).

Reply:
I guess it comes down to what you mean by "production". In my view, the production stuff is what will be deployed to the web server. To be honest, since we are not using `npm` to do the deployment, it is irrelevant. I would be happy to move everything into either `dependencies` or `devDependencies` but I feel that putting the angular libraries (and the html5-boilerplate) into the dependencies, it is signalling to the developer what will eventually end up as part of the actually deployed application.